styles/dogs.css

```diff
@@ -0,0 +1,7 @@
+.dogs {
+    display: grid;
```

Comment:
Watch your indentation here, looks like 4 spaces rather than the two you use in the other files